app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_store-scope.less

```diff
@@ -6,20 +6,18 @@
 .tree-store-scope {
     &.admin__fieldset {
         padding: 0;
-
-        > .admin__field {
-            margin-left: -@temp_gutter;
-        }
     }
 
     [class*='field-w'],
     [class*='field-sg'] {
         &.admin__field {
-            margin: 0 0 0 -@temp_gutter;
+            margin: 0 0 0 -30px;
```

Comment:
Hi @suryakant-krish. We prefer to use variables to have the same sizes in all places. Please use one of the existing variables